styleSheet.css

```diff
@@ -236,6 +236,58 @@ ul {
   width: 100%;
 }
 
+#awardsInterest {
+  background-color: var(--dark__primary);
+  font-size: 1.2em;
+  height: calc(60vh + 20vw);
+  display: inline-flex;
+}
+
+.awardsInterest__list {
+  background-color: var(--light__primary);
+  border-radius: 20px;
+  height: 90%;
+  margin: 2%;
+  padding: 0 10px;
+  position: initial;
+  width: 85%;
+  z-index: 20;
+}
+
+.awardsInterest__list > h3 {
+  font-size: calc(2rem + 0.5vw);
+  margin-bottom: 0%;
```

Comment:
I think you can avoid the percentage here. Just `margin-bottom: 0`. Same in all the other cases

Reply:
Yeah, I will keep that to mind.